README.md

```diff
@@ -427,15 +426,60 @@ RULE:     - the owning team will provide timely, high quality updates for the
 RULE:       security team to sponsor to fix issues in the affected vendored code
 RULE:     - if subsequent uploads add new vendored components or dependencies
 RULE:       these have to be reviewed and agreed by the security team.
+RULE:     - Such updates in the project might be trivial, but imply that a
+RULE:       dependency for e.g. a CVE fix will be moved to a new major version.
+RULE:       Being vendored that does gladly at least not imply incompatibility
+RULE:       issues with other packages or the SRU policy. But it might happen
+RULE:       that this triggers either:
+RULE:       a) The need to adapt the current version of the main package and/or
+RULE:          other vendored dependencies to work with the new dependency
+RULE:       b) The need to backport the fix in the dependency as the main
+RULE:          package will functionally only work well with the older version
+RULE:       c) The need to backport the fix in the dependency, as it would imply
+RULE:          requiring a newer toolchain to be buildable that isn't available
+RULE:          in the target release.
+RULE: - The rust ecosystem currently isn't yet considered stable enough for
+RULE:   classic lib dependencies and transitions in main; therefore the
+RULE:   expectation for those packages is to vendor (and own/test) all
+RULE:   dependencies (except those provided by the rust runtime itself).
+RULE:   This implies that all the rules for vendored builds always
+RULE:   apply to them. In addition:
+RULE:   - The rules and checks for rust based packages are preliminary and might
+RULE:     change over time as the ecosytem matures and while
+RULE:     processing the first few rust based packages.
+RULE:   - It is expected rust builds will use dh-cargo so that a later switch
+RULE:     to non vendored dependencies isn't too complex (e.g. it is likely
+RULE:     that over time more common libs shall become stable and then archive
+RULE:     packages will be used to build).
+RULE:   - the rust tooling e.g. dh-cargo can not yet automatically provide
+RULE:     all that we require - for example Cargo.lock - until it does the
```

Comment:
I'm very concerned that adopting these rules before we have a good `Cargo.lock` story will create a very difficult situation to cope with:

- At some point, we will need to provide SBOMs to sell to specific clients and into specific industries.
- Having detailed summarized information available short of "search all the source code" is a basic necessity for maintaining these packages ten years from now.

While I'm overall a big fan of the "get something working and then iterate on it as needs are identified", I'm also very scared that we'll wind up with dozens of packages and no tooling to track the vendored dependencies.

If we let in one, the arguments against the second, third, and fourth get increasingly weak.

I believe we need a solution to this problem before we let in the first package.

Thanks